File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_01_Lecture/_02_PoundsToDollars.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._01_Lecture;

import java.util.Scanner;

public class _02_PoundsToDollars {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        double inputPounds = Double.parseDouble(scan.nextLine());

        // Add variable
        double poundsToDollars = inputPounds * 1.36;

        // Print value
        System.out.printf("%.3f", poundsToDollars);

        scan.close();
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_01_Lecture/_02_PoundsToDollars/_02_PoundsToDollars.cs
using System;

public class _02_PoundsToDollars
{
    public static void Main(string[] args)
    {
        // Read input value
        double inputPounds = double.Parse(Console.ReadLine());

        // Add variable
        double poundsToDollars = inputPounds * 1.31;

        // Print result
        Console.WriteLine($"{poundsToDollars:F3}", poundsToDollars);
    }
}
